src/components/CommentsLoading.jsx

```diff
@@ -0,0 +1,40 @@
+import { useState, useEffect } from "react";
+import { getComments } from "../../api";
+import { useParams } from "react-router-dom";
+
+function CommentsLoading() {
+  const [comments, setComments] = useState([]);
+  const [isLoading, setIsLoading] = useState(true);
+  const { article_id } = useParams();
+
+  useEffect(
+    () => {
```

Comment:
don't forget to set isLoading back to true at the beginning of the useEffect in case an error occurs and requires re-rendering